test/e2e/image_pull_error_test.go

```diff
@@ -86,7 +88,7 @@ func TestImagePullError(t *testing.T) {
 		cond := r.Status.GetCondition(v1alpha1.RevisionConditionReady)
 		if cond != nil {
 			if (cond.Reason == backoffReason && strings.Contains(cond.Message, backoffMsg)) ||
-				(cond.Reason == daemonReason && strings.Contains(cond.Message, daemonMsg)) {
+				(cond.Reason == errReason && (strings.Contains(cond.Message, errMsg1) || strings.Contains(cond.Message, errMsg2))) {
```

Comment:
We could save the error message from above and make sure the same error message gets propagated here, if we think that's worth checking.